ember-server/shared/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala

```diff
@@ -75,31 +76,40 @@ private[server] object ServerHelpers extends ServerHelpersPlatform {
       webSocketKey: Key[WebSocketContext[F]],
       enableHttp2: Boolean,
   )(implicit F: Async[F]): Stream[F, Nothing] = {
-    val server: Stream[F, Socket[F]] =
-      Stream
-        .resource(sg.serverResource(host, Some(port), additionalSocketOptions))
-        .attempt
-        .evalTap(e => ready.complete(e.map(_._1)))
-        .rethrow
-        .flatMap(_._2)
-    serverInternal(
-      server,
-      httpApp: HttpApp[F],
-      tlsInfoOpt: Option[(TLSContext[F], TLSParameters)],
-      shutdown: Shutdown[F],
-      // Defaults
-      errorHandler: Throwable => F[Response[F]],
-      onWriteFailure: (Option[Request[F]], Response[F], Throwable) => F[Unit],
-      maxConnections: Int,
-      receiveBufferSize: Int,
-      maxHeaderSize: Int,
-      requestHeaderReceiveTimeout: Duration,
-      idleTimeout: Duration,
-      logger: Logger[F],
-      true,
-      webSocketKey,
-      enableHttp2,
-    )
+    val res = Resource.uncancelable[F, Unit] { _ =>
```

Comment:
Upvote this question 👍🏻

Reply:
So sorry that I'm only getting round to this now. I think I put it in because of the `allocated` but actually resource acquisition (`Resource.make`) is uncancelable anyway, right?

Reply:
Yes, `make` is uncancelable.